plugins/search-backend-module-elasticsearch/package.json

```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "@backstage/config": "^0.1.13",
     "@backstage/search-common": "^0.2.0",
-    "@elastic/elasticsearch": "7.13.0",
+    "@elastic/elasticsearch": "7.16.0",
```

Comment:
Given the breaking change this pinning seems wise :grin:

Is it worth switching to `~` tho?

Reply:
Anything past `7.13.x` doesn't work with AWS hosted 'OpenSearch' so we'd need to keep the base at that one until we find a better solution to support both Elastic Elastic- and AWS OpenSearches. `7.13.0` is the only version on that minor number so in reality it doesn't make a difference if we add a tilde or not. I doubt new patches for that minor will be published in the future.

Reply:
Yep, just thinking as a pattern. I'm feeling `~` conveys more intent as this might be mistaken for a mistake :grin: